pkg/cloud-provider/cloudapi/aws/aws_ec2.go

```diff
@@ -40,17 +41,21 @@ type ec2ServiceConfig struct {
 	//	 - key with nil value indicates no filters. Get all instances for account.
 	//   - key with "some-filter-string" value indicates some filter. Get instances matching those filters only.
 	instanceFilters map[string][][]*ec2.Filter
+	credentials     *awsAccountConfig
+	pollVpc         bool
```

Comment:
pollVpc seems futuristic and currently its not serving any purpose and its also not the right place to add. Please cleanup the code for this pollVPC